lib/matplotlib/table.py

```diff
@@ -558,6 +590,10 @@ def table(ax,
           rowLabels=None, rowColours=None, rowLoc='left',
           colLabels=None, colColours=None, colLoc='center',
           loc='bottom', bbox=None, edges='closed',
+          cellEdgeColour=None,
```

Comment:
Please do not add both the plural and singular versions, use the plural and let it take a single value (which is implicitly broadcast) or an iterable.  The function `matplotlib.colors.to_rgba_array` takes care of most of this work for you.

The library has settled on the American spelling of color, but I see that this signature already uses the British spelling, so stick with that.